C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -408,7 +441,8 @@ int Connection::readingStream(ReadingStream **readings, bool commit)
 	struct timeval start, t1, t2, t3, t4, t5;
 #endif
 
-	const char *sql_cmd = "INSERT INTO  " DB_READINGS ".readings ( asset_code, reading, user_ts ) VALUES  (?,?,?)";
+	// * TODO: the current code should be adapted to use the multi databases/tables implementation
```

Comment:
Let's raise a Jira for this so we don't forget. I can imagine the streaming will come back into importance if this uncovers that as the throttle point now.

Reply:
done, FOGL-4457